eo-maven-plugin/src/main/java/org/eolang/maven/name/OnReplaced.java

```diff
@@ -67,13 +70,33 @@ public final class OnReplaced implements ObjectName {
      * - "org.eolang.string#1.23.1"
      * - "org.eolang.math#3.3.3"
      */
-    private final String raw;
+    private final Unchecked<String> raw;
```

Comment:
@maxonfjvipon I don't see the reason for that change. Can we keep that code [simple](https://en.wikipedia.org/wiki/KISS_principle)?

Reply:
@volodya-lombrozo please check code [here](https://github.com/objectionary/eo/pull/2398/files#r1294913645)